config/examples/redis.yml

```diff
@@ -3,8 +3,20 @@ base: &default
   size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   pool_timeout: 5 # this is in seconds
   sentinels: "<%= ENV['REDIS_SENTINEL_HOSTS'] %>"
+  sentinel_username: <%= ENV['REDIS_SENTINEL_USERNAME'].to_json %>
```

Comment:
I haven't tested these, but I guess it should just work, as we just pass the config to the redis client.

Reply:
I tried locally and it works